### PR TITLE
Add essay about math

### PR DESCRIPTION
**Pull request by salutis**
A full 5-paragraph essay! No research.

### CODE REVIEW

File: Math.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+
+Unlike other fields, math is different in that it ties all of our knowledge together. It is the shared language of modern science. Whether the student is working towards their physics, business or a computer science degree, math is used as the common language, the unified, symbolic way of communication. This enables cross-pollination between scientific fields which ultimately leads to progress.
+
+In addition to enabling progress of all science, math is also useful in virtually all areas of everyday life. Whether a person is buying a new house, struggling with personal finances or planning retirement, math provides clear, unambiguous and verifiable answers to many questions that need to be answered to make good decisions.
+
+Finally, math knowledge helps one to think clearly about the big questions of life. For instance, consider charlatans promising everything under the sun, from happiness and wealth, to the eternal life. Math teaches us how to think correctly about them. It gives us tools to evaluate truthiness of their arguments as well as our own thoughts. It allows us to debunk myths and focus on the objective reality around us instead.
```
Comment (mikemee) [inline Math.txt]: to ~~the~~ eternal life

Comment (mikemee) [reply Math.txt]: truthiness -> the truthfullness  (not sure truthiness is a word...)

**Comment by mikemee** [comment]
This is excellent!  And you have such a large vocabulary.

At this rate of progress, you will breeze through the English course. Keep it up!

File: Math.txt (hunk 1 above)
Comment (salutis) [reply Math.txt]: Oh! Just looked it up on the Wikipedia and it's exactly what I did **not** want to say (emphasis mine):

> Truthiness is the belief or assertion that a particular statement is true based on the intuition or perceptions of some individual or individuals, **without regard to evidence, logic, intellectual examination, or facts**.

English is tricky!

**Comment by salutis** [comment]
Review feedback addressed, merging. Thanks @mikemee!

File: Math.txt [hunk 2]
```diff
@@ -4,6 +4,6 @@ Unlike other fields, math is different in that it ties all of our knowledge toge
 
 In addition to enabling progress of all science, math is also useful in virtually all areas of everyday life. Whether a person is buying a new house, struggling with personal finances or planning retirement, math provides clear, unambiguous and verifiable answers to many questions that need to be answered to make good decisions.
 
-Finally, math knowledge helps one to think clearly about the big questions of life. For instance, consider charlatans promising everything under the sun, from happiness and wealth, to the eternal life. Math teaches us how to think correctly about them. It gives us tools to evaluate truthiness of their arguments as well as our own thoughts. It allows us to debunk myths and focus on the objective reality around us instead.
+Finally, math knowledge helps one to think clearly about the big questions of life. For instance, consider charlatans promising everything under the sun, from happiness and wealth, to eternal life. Math teaches us how to think correctly about them. It gives us tools to evaluate truthfullness of their arguments as well as our own thoughts. It allows us to debunk myths and focus on the objective reality around us instead.
```
Comment (mikemee) [inline Math.txt]: *the* truthfullness - sorry if I missed that in the last comment

Comment (salutis) [reply Math.txt]: Fixed in #13, thank you!

**Pull request merged**